src/services/LoggingService.js

```diff
@@ -4,7 +4,9 @@ import log4js from 'log4js';
 dotenv.config();
 
 const getLogger = (key) => {
-  const logger = key ? log4js.getLogger(`BPP-${key}`) : log4js.getLogger('GatewayRegistry');
+  const logger = key
+    ? log4js.getLogger(`BPP-${key}`)
+    : log4js.getLogger('GatewayRegistry');
```

Comment:
Make this BPP; Its a existing copy/paste bug;
log4js.getLogger('GatewayRegistry');
should be log4js.getLogger('BPP')

Reply:
Updated